Sources/Datadog/RUM/RUMMonitor/Scopes/RUMResourceScope.swift

```diff
@@ -95,8 +104,12 @@ internal class RUMResourceScope: RUMScope {
             ),
             usr: dependencies.userInfoProvider.current,
             connectivity: dependencies.connectivityInfoProvider.current,
-            dd: .init(),
+            dd: .init(
+                spanID: spanContext.flatMap { String($0.spanID) },
+                traceID: spanContext.flatMap { String($0.traceID) }
+            ),
```

Comment:
Indeed, the `TracingUUID` → `Int64` → `String` mapping was unnecessary. I updated it to do `TracingUUID` → `String` directly in `URLSessionRUMResourcesHandler`.

> as a bridge, RUMSpanContext can also provide `init(from: DDSpanContext)` and `var toResourceDD { ... }` helpers (otherwise obviously every caller needs to know how to convert)

`init(from: DDSpanContext)` - I haven't done it intentionally, to not introduce more coupling between Tracing and RUM features. Now, the coupling happens only in `URLSessionRUMResourcesHandler` (for mapping `Tracing.DDSpanContext` to `RUM.RUMSpanContext`). It's simple and easy embrace it now and refactor if needed - spreading it more into RUM codebase will make both harder.

`var toRUMDataFormat` - I've been thinking of it, but it doesn't fit in this particular case. What we have is `RUMSpanContext`, what we'd need from `.toRUMDataFormat` is `RUMResourceDD` which describes more than just span context. Let's wait and see how it evolves, then we can make this small refactor.

> by the way, i can't find occurences of any type with `.init` usage with Find selected text/symbol in Workspace; how do you search?

⌘ + ⇧ + Click on the symbol, and here you go:

<img width="851" alt="Screenshot 2020-10-22 at 13 18 11" src="https://user-images.githubusercontent.com/2358722/96864664-13214080-1469-11eb-8884-c50bb7aa32b1.png">